pkg/webhooks/machine_webhook.go

```diff
@@ -110,6 +111,13 @@ const (
 	minVSphereMemoryMiB = 2048
 	// https://docs.openshift.com/container-platform/4.1/installing/installing_vsphere/installing-vsphere.html#minimum-resource-requirements_installing-vsphere
 	minVSphereDiskGiB = 120
+
+	// PowerVS Defaults
+	defaultPowerVSCredentialsSecret = "powervs-credentials"
+	defaultPowerVSSysType           = "s922"
+	defaultPowerVSProcType          = "shared"
+	defaultPowerVSProcessor         = "0.25"
+	defaultPowerVSMemory            = "8"
```

Comment:
@clnperez need your help deciding what should be the right values here for any worker ^^

Reply:
@clnperez shall I go ahead with this https://github.com/openshift-powervs/installer/blob/2e17e5afefda5976366d71caa7cdee03fa42ceec/pkg/asset/machines/worker.go#L165  as default values.

Reply:
i would recommend the values the UPI folks have been using: https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/7279803eed6566faf820bcac1334dc8e7f5ba761/compute-vars/medium.tfvars#L6